fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -2492,6 +2494,38 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 		return EncryptionAtRestMode(EncryptionAtRestMode::DISABLED);
 	}
 
+	Future<Void> ingestSSTFiles(std::string bulkLoadLocalDir,
```

Comment:
Seems that `bulkLoadLocalDir` is not using?

Reply:
Good find. It was added in this patch way back. It was only being used in trace events but they have now been removed.  Let me fix.